command/agent/consul/connect_test.go

```diff
@@ -424,7 +424,29 @@ func TestConnect_connectProxyConfig(t *testing.T) {
 			"envoy_stats_tags": []string{"nomad.alloc_id=test_alloc2"},
 		}, connectProxyConfig(map[string]interface{}{
 			"foo": "bar",
-		}, 42, structs.AllocInfo{AllocID: "test_alloc2"}))
+		}, 42, structs.AllocInfo{AllocID: "test_alloc2"}, nil))
+	})
+
+	t.Run("bind_address override", func(t *testing.T) {
+		require.Equal(t, map[string]interface{}{
```

Comment:
We should probably try to avoid adding new tests using `testify` instead of `shoenig/test`